generation/maisi/configs/config_maisi.json

```diff
@@ -120,11 +120,11 @@
         "dim_split": 1
     },
     "mask_generation_diffusion_def": {
-        "_target_": "monai.apps.generation.maisi.networks.diffusion_model_unet_maisi.DiffusionModelUNetMaisi",
+        "_target_": "monai.networks.nets.diffusion_model_unet.DiffusionModelUNet",
```

Comment:
May I know why we change the network here? cc @dongyang0122

Reply:
It was written by @guopengf . It should be DiffusionModelUNet

Reply:
Correct, the mask generation model is not a customized diffusion unet